e2e-tests/portal/cypress/e2e/journeys/maker-bond/bond-form-values.js

```diff
@@ -1,13 +1,26 @@
-const { roundNumber } = require('../../../../../../portal-api/src/utils/number');
-const { oneMonth, today } = require('../../../../../e2e-fixtures/dateConstants');
+import { BOND_FACILITY_TYPE } from '@ukef/dtfs2-common';
+import { oneMonth, today } from '../../../../../e2e-fixtures/dateConstants';
 
-const DETAILS = {
+const roundNumber = (value, digits) => {
```

Comment:
Would be good to get some jsdocs here - probably saying where this was taken from & what its actually doing? 
e.g. why is this the logic?
```
  if (!digits) {
    d = 2;
  }
  ```

Reply:
This seems like an interesting method -- It scales up the number by the power of the required digits, and then divides. Really not sure why round cannot be used (Could quickly swap the internal stuff of this function out with round and see if it passes...)

Reply:
@AlexBramhill Will have a play once I can run the test on my machine...

Reply:
I had a play - it's not straight forward to replace or improve this. 

I have added a new ticket to the backlog to improve this: EMS-4002

This PR has been around for a while, don't want to make this any larger.

Reply:
Very wise. Sound good to you @MarRobSoftwire?